apps/trading-e2e/src/support/pages/deposits-page.ts

```diff
@@ -49,6 +54,12 @@ export default class DepositsPage extends BasePage {
       .should('be.visible');
   }
 
+  veryInsufficientAmountAmountMessage() {
+    cy.getByTestId('input-error-text')
+      .contains('Insufficient amount in Ethereum wallet')
+      .should('be.visible');
```

Comment:
I don't think `.contains` is needed here. The text should be asserted by the assertion:
`cy.getByTestId('input-error-text').should('have.text', 'Insufficient amount in Ethereum wallet')

Reply:
_input-error-text_ is present multiple times and have.text will fail , include.text should work in this case but then does the same thing as above